src/docs/styles/custom.css

```diff
@@ -68,3 +68,20 @@ body,
 #app table td {
     text-align: center;
 }
+
+/** Loading container */
+.loading-container {
+    position: relative;
+}
+
+.loading-container .load-state {
+    width: 100%;
+    height: 100%;
+    top: 0;
+    left: 0;
+    position: absolute;
+}
+
+.loading-container--loading .britechart {
+    visibility: hidden;
+}
```

Comment:
Oops
It seems we would need to create a britecharts-react stylesheet, generate it on dist/ and ask the users to load it.
This custom stylesheet is only for the demos.

I think this could make us rethink the approach. Maybe it would make sense to show the loading state only as part of britecharts. Let's talk about it in person.